src/server.spec.ts

```diff
@@ -173,9 +173,9 @@ describe("Server Spec", () => {
 			});
 
 			it("should start correctly with monkeypatch", () => {
-				const server = serverFactory({monkeypatch: monkeypatchFile});
-				expect(server.options.monkeypatch).to.equal(monkeypatchFile);
-				return expect(server.start()).to.eventually.be.fulfilled;
+				const s = serverFactory({monkeypatch: monkeypatchFile});
```

Comment:
Could have disabled a linting rule here (`no-shadow`) but felt this was less evil.

Reply:
:+1: